jenkins/vulnerability-scan/whitesource-scan.jenkinsfile

```diff
@@ -44,9 +44,9 @@ pipeline {
 
 void scan() {
     sh """
-        curl -SLO https://services.gradle.org/distributions/gradle-6.6.1-bin.zip
-        unzip gradle-6.6.1-bin.zip
-        GRADLE_HOME=`realpath gradle-6.6.1`
+        curl -SLO https://services.gradle.org/distributions/gradle-7.4.2-bin.zip
```

Comment:
Yes from our latest Jenkins WS scan I see this kind of error on scanning OpenSearch repo
```
[WARN] [2022-07-01 03:18:01,879 +0000] - Read error line #5: A problem occurred evaluating project ':buildSrc'.
[WARN] [2022-07-01 03:18:01,879 +0000] - Read error line #6: > Gradle 7.4.1+ is required to build opensearch
```
Updating the gradle version here should helps.